### ISSUE TITLE
yolov1训练时loss为nan

### ISSUE THREAD

**Issue by lixiaoming0017** [bobo0810/PytorchNetHub#12]
您好，我在用yolov1的代码时候，用main.py训练，数据集合只用了VOC2007的数据，pytorch用的1.2，修改了total_loss += loss.data[0],改为total_loss += loss.item(),训练的时候loss一直是nan,别的参数没改，请问一下这是什么原因啊

**Comment by boomboomgwx** [comment]
我和你一样，不知道为啥
